packages/react-input/src/stories/InputAppearance.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { Label } from '@fluentui/react-label';
+import { useId } from '@fluentui/react-utilities';
+import { makeStyles } from '@fluentui/react-make-styles';
+import { Input } from '../index';
+
+const useStyles = makeStyles({
+  root: {
+    '& label': { display: 'block', paddingBottom: '2px' },
+    '& label:not(:first-child)': { paddingTop: '20px' },
+  },
+});
```

Comment:
I wasn't sure how we're handling things like this for stories in vnext: should we optimize for making them standalone (when they're copy-pasted or exported to codesandbox, it will look identical to the example), or for making them simple to read? The disadvantage of hiding the styling for label-input spacing in particular is that people will think it's automatic, when actually it's not at the moment.